doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-test-scenario-export-spreadsheet-proc.adoc

```diff
@@ -0,0 +1,10 @@
+[id='test-designer-test-scenario-export-spreadsheet-proc']
+= Exporting a test scenario spreadsheet
+
+Follow the procedure below to export a test scenarios spreadsheet data using the Test Scenario designer.
+
+.Procedure
+. From the Test Scenario designer toolbar on the upper-right, click *Export* image:AuthoringAssets/test-scenarios-export-button.png[] button.
+. Select a destination for the file in your local file system and confirm the choice to save the `.csv` file generated from the Test Scenario designer.
```

Comment:
Select a destination for the file in your local file system and confirm the choice to save the `.csv` file generated from the Test Scenario designer.

__1__ a destination in your local file system and __2__

1 "Navigate to" or "Select"
2 "click *Save*.", "click *Confirm*.", or "confirm to save the `.csv` file."

Reply:
Ack

Reply:
Done.

Reply:
From the Test Scenario designer or In the Test....?

According to the IBM Style guide (pg 208-215) panes, portals, windows, wizards use the verb "in".

"local directory' not file system. And how does one confirm? Is there a "Confirm" button?